sdk/storage/storage-blob/test/utils/index.ts

```diff
@@ -55,6 +56,7 @@ export function getGenericBSU(
     const credential = getGenericCredential(accountType) as StorageSharedKeyCredential;
 
     const pipeline = newPipeline(credential, {
+      ...pipelineOptions
```

Comment:
>...pipelineOptions [](start = 6, length = 18)

By default we enabled keep-alive right? Is this change required to reproduce the bug in UT?

Reply:
Not sure. And do not want to depend on core-http for setting it.